spec.bs

```diff
@@ -296,9 +296,11 @@ This is detectable because it can change the set of fields that are read from th
   1. If |group|["{{GenerateBidInterestGroup/trustedBiddingSignalsKeys}}"] [=map/exists=], then set
     |interestGroup|'s [=interest group/trusted bidding signals keys=] to
     |group|["{{GenerateBidInterestGroup/trustedBiddingSignalsKeys}}"].
-  1. If |group|["{{GenerateBidInterestGroup/maxTrustedBiddingSignalsURLLength}}"]'s value is non-negative,
-    set |interestGroup|'s [=interest group/max trusted bidding signals url length=] to
-    |group|["{{GenerateBidInterestGroup/maxTrustedBiddingSignalsURLLength}}"], otherwise [=exception/throw=] a {{TypeError}}.
+  1. If |group|["{{GenerateBidInterestGroup/maxTrustedBiddingSignalsURLLength}}"] [=map/exists=]

```

Comment:
we don't throw when the field does not exist.
So you need to separate the if statements:
"
If ... [=map/exists=] {
if ... >=0, then set ..., otherwise throw ...
}
"